hikari/api/interaction_server.py

```diff
@@ -87,7 +87,7 @@ def charset(self) -> typing.Optional[str]:
         raise NotImplementedError
 
     @property
-    def files(self) -> typing.Sequence[files_.Resource[files_.AsyncReader]]:
+    def files(self) -> typing.Mapping[int, files_.Resource[files_.AsyncReader]]:
```

Comment:
This and `InteractionResponseBuilder.build`'s return types are the only breaking changes in this PR